File: java/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/event_listener/BannerNativeAdEventListener.java
package com.pubscale.sdkone.example.abtestexample.event_listener;

public interface BannerNativeAdEventListener {
    void onAdLoading();
    void onAdLoaded();
    void onAdLoadFailed();
    void onUiiOpened();
    void onUiiClosed();
    void onReadyForRefresh();
}

File: kotlin/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/event_listener/BannerNativeAdEventListener.kt
package com.pubscale.sdkone.example.abtestexample.event_listener

interface BannerNativeAdEventListener {
    fun onAdLoading()
    fun onAdLoaded()
    fun onAdLoadFailed()
    fun onUiiOpened()
    fun onUiiClosed()
    fun onReadyForRefresh()
}